bin/build.sh

```diff
@@ -668,10 +671,8 @@ install()
 # Start of 'main'
 #
 
-utildir=$( cd $(dirname "$0")/ || exit; echo $PWD)
-export utildir
-utilparentdir=$( cd $(dirname "$0")/../ || exit; echo $PWD)
-export utilparentdir
+export utildir="$( cd "$(dirname "$0")" >/dev/null 2>&1 && pwd -P )"
```

Comment:
i originally had something like this (minus the && pwd -P) and a shell script scanner told me I should be checking the cd succeeded and that's why i had had two steps... I'm ok either way. What's the advantage of pwd -P vs echo $PWD?